src/Microsoft.AspNet.Mvc.Core/MvcOptionsExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc
+{
+    public static class MvcOptionsExtensions
+    {
+        /// <summary>
+        /// Adds <see cref="XmlDataContractSerializerInputFormatter"/> and <see cref="XmlDataContractSerializerOutputFormatter"/>
```

Comment:
this line is longer than 120 characters ... hence it causes to show up here with a horizontal scroll bar, we have a style cop rule which asks us to keep line lengths below 120. Line break means break the line into two lines so that the length remains under 120.